### PR TITLE
Update changelog to version 2.0.0

### PR DESCRIPTION
**Pull request by DabeerAzeez**
- Changed the structure of the `changelog.json` file to include a boolean to tell whether a version will allow users to send a new message to each of their friends. Also added frontend to display this information within the changelog modal.

### CODE REVIEW

**Review by ricetech — CHANGES_REQUESTED** [review]
Blocked by #422

File: src/app/core/navbar/navbar.component.html [hunk 1]
```diff
@@ -167,10 +167,10 @@ <h5 class="modal-title" id="changelog-label">Changelog</h5>
...
             >Note: Whenever we release an update to the 'signing' page, we may,
             at our discretion, allow you to send <em>another</em> message to
-            each of your friends' virtrolios! This ensures that you will be able
-            to enjoy any new features that we develop. Keep an eye on this
-            changelog as we will mention here if we ever do give you another
-            message opportunity.</small
+            each of your friends' virtrolios as if it were a new year! This
+            ensures that you will be able to enjoy any new features that we
+            develop. Keep an eye out for versions with a 💌 emoji next to them
+            to see if you're allowed to spread some more Virtrolio joy!</small
```
Comment (ricetech) [inline src/app/core/navbar/navbar.component.html]: I like the wording before but we also do need to be extra clear to make sure people don't miss it
```suggestion
            to see if you're allowed to send an extra message!</small
```

File: src/app/core/navbar/changelog.json [hunk 2]
```diff
@@ -1,4 +1,61 @@
...
+        "betaFlag": false,
+        "location": "Viewing",
+        "content": "Replace the crying emoji with more professional content and graphics when there are no messages to view \ud83d\ude2d"
+      },
+      {
+        "type": "NEW",
+        "typeCSS": "changelog-new",
+        "betaFlag": false,
+        "location": "Sitewide",
+        "content": "Update content and graphics on static pages"
```
Comment (ricetech) [inline src/app/core/navbar/changelog.json]:
```suggestion
        "content": "Revamp the graphics used throughout the website"
```

**Review by ricetech — CHANGES_REQUESTED** [review]
Missing changelog item - spinner while signing

**Review by kishanmanoharan — APPROVED** [review]

**Review by ricetech — APPROVED** [review]

**Pull request merged**